types/encode.go

```diff
@@ -25,15 +27,15 @@ func BlockIdsAsBytes(ids []BlockID) ([]byte, error) {
 func BytesToBlockIds(blockIds []byte) ([]BlockID, error) {
 	var ids []BlockID
```

Comment:
we should use the same naming convention for all conversion methods,
i think it should be ViewToBytes and not ViewAsBytes